projects/objects/balls/protos/Ball.proto

```diff
@@ -5,8 +5,8 @@
 # You can change the centerOfMass to [0 0 0] to avoid the random rolling effect.
 
 PROTO Ball [
-  field SFVec3f    translation     0 0.0325 0
-  field SFRotation rotation        0 1 0 0
+  field SFVec3f    translation     0 0 0
+  field SFRotation rotation        0 0 1 0
```

Comment:
Solid default rotation value is `0 1 0 0`.
Is there a particular reason to set a different default rotation for balls?

Reply:
It's not very important, but means that the balls are now compatible with the [new default ENU orientation recommendations](https://cyberbotics.com/doc/reference/proto-design-guidelines#orientation).

Reply:
Webots will soon parse this field to determine if a PROTO has to be rotated when inserted from the GUI in a non-ENU world, but for balls, it doesn't make any sense to rotate them 😉.

Reply:
Again, I think we should be consistent with default rotation value.
For me it is ok to set this here only if we will also change the `Solid` default rotation in R2021b, otherwise it doesn't make sense.

Reply:
Note that `0 1 0 0` and `0 0 1 0` are both empty rotations. It is just a matter of consistency between default values in the models distributed within Webots.

Reply:
Yes, the plan is also to change the default rotation in Solid as well, but this is out of the scope of this PR.